src/main/scala/coreplex/Coreplex.scala

```diff
@@ -65,21 +80,82 @@ abstract class Coreplex(implicit val p: Parameters, implicit val c: CoreplexConf
 }
 
 class DefaultCoreplex(tp: Parameters, tc: CoreplexConfig) extends Coreplex()(tp, tc) {
+  class TileWrapper(tileBuilder: (Bool, Parameters) => Tile, _clock: Clock, _reset: Bool)
```

Comment:
Are you suggesting that I leave the instantiations of the tile and L2 as methods and then override those in a MultiClockCoreplex subclass? What if I want to multiclock the tiles, but not the L2 or vice-versa?